test/IDE/print_clang_objc_async.swift

```diff
@@ -124,3 +124,20 @@ import _Concurrency
 // CHECK: func doSomethingConcurrently(_ block: @Sendable () -> Void)
 
 // CHECK: @MainActor @objc protocol TripleMainActor {
+
+// CHECK-LABEL: class NXSender :
+// CHECK-NEXT: func sendAny(_ obj: Sendable)
+// CHECK-NEXT: func sendOptionalAny(_ obj: Sendable?)
+// CHECK-NEXT: func sendSendable(_ sendable: SendableClass & Sendable)
+// CHECK-NEXT: func sendSendableSubclasses(_ sendableSubclass: NonSendableClass & Sendable)
+// CHECK-NEXT: func sendProto(_ obj: LabellyProtocol & Sendable)
+// CHECK-NEXT: func sendProtos(_ obj: LabellyProtocol & ObjCClub & Sendable)
+// CHECK-NEXT: func sendAnyArray(_ array: [Sendable])
+// CHECK-NEXT: func sendGeneric(_ generic: GenericObject<SendableClass> & Sendable)
+// CHECK-NEXT: func sendPtr(_ val: UnsafeMutableRawPointer)
```

Comment:
Should there be a warning about dropping the explicit sendability from the ObjC decl here and for the next function decl?

Reply:
I do emit some warnings through the experimental ClangImporter diagnostic infrastructure, but I don't want to rope that into these tests at its current stage of development.